packages/apps/docs/src/utils/createSlug.ts

```diff
@@ -1,10 +1,21 @@
-export const createSlug = (str?: string): string => {
+export const createSlug = (str?: string, index?: number): string => {
   if (str === undefined) return '';
-  return str
+  const normalizedSlug = str
     .normalize('NFD')
     .replace(/[\u0300-\u036f]/g, '')
     .replace(/[^\w\s]+/g, '')
     .replace(/ /g, '-')
     .toLowerCase()
     .replace(/^-+|-+$/g, '');
+
+  if (normalizedSlug === '' && index !== undefined) return `operator-${index}`;
```

Comment:
could we make `operator` more generic? now its just there for operators.
maybe just `header-idx`

Reply:
how about `aside-menu-idx` or `menu-idx` ? because header it's more of the component where it's used but we need this value for link reference